deepchecks/nlp/utils/embeddings_display.py

```diff
@@ -0,0 +1,163 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils module for calculating embeddings display checks."""
+
+import os
+import time
+from typing import Dict, List
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from PyNomaly import loop
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.metrics import log_loss, roc_auc_score
+from sklearn.model_selection import train_test_split
+
+from deepchecks.core.check_utils.multivariate_drift_utils import auc_to_drift_score, build_drift_plot
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.nlp.utils.embeddings_calculator import clean_special_chars
+from deepchecks.utils.dataframes import floatify_dataframe
+
+
+def create_performance_display(text: pd.Series, embeddings: np.ndarray, proba: np.ndarray, y_true: np.ndarray,
```

Comment:
A bit of nirpicking, but maybe the names should be create_*_files and not display (because it doesn't create the display)

Reply:
Fine by me, will change